internal/template.go

```diff
@@ -68,6 +61,11 @@ func GetFuncMap(profile string) template.FuncMap {
 		}
 		return str, err
 	}
+
+	funcMap["quote"] = func(input string) string {
```

Comment:
I think we only use quote from the engine funcMap, which is easy enough to implement